packages/core/src/config/parse.ts

```diff
@@ -1077,21 +1194,27 @@ export const assertValidParsedChugSplashFile = async (
   artifactPaths: ArtifactPaths,
   integration: Integration,
   cre: ChugSplashRuntimeEnvironment
-) => {
+): Promise<{
+  valid: boolean
+  upgrade: boolean
+}> => {
   const {
     canonicalConfigPath,
     remoteExecution,
-    autoConfirm,
-    openzeppelinStorageLayouts,
+    fetchOpenZeppelinStorageLayouts,
   } = cre
 
-  // Determine if the deployment is an upgrade
-  const projectName = parsedConfig.options.projectName
+  let isValid = true
+
+  const openzeppelinStorageLayouts = await fetchOpenZeppelinStorageLayouts(
+    cre.hre,
+    userConfig
+  )
```

Comment:
Is there a reason we include the `hre` inside of the `cre` now? IMO it's more straightforward to call `importOpenZeppelinStorageLayouts` inside of `createChugSplashRuntime`, which is how we did this previously. I think it's a cleaner approach, plus we don't need to pass around the HRE inside of the core package